src/category_theory/products/associator.lean

```diff
@@ -7,6 +7,12 @@ import category_theory.products
 import category_theory.equivalence
 import category_theory.eq_to_hom
 
+/-#
+The swap functor `C × D ⥤ D × C` is an equivalence.
+
+The associator functor `((C × D) × E) ⥤ (C × (D × E))` and its inverse form an equivalence.
```

Comment:
If you have the braiding and associator, I guess it might make sense to also include unitors...

Reply:
I've added a comment noting that unitors have not been done yet.